pkg/publish/layout.go

```diff
@@ -25,22 +25,27 @@ import (
 	"github.com/google/go-containerregistry/pkg/v1/layout"
 	"github.com/google/go-containerregistry/pkg/v1/types"
 	"github.com/google/ko/pkg/build"
+	specsv1 "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
 type LayoutPublisher struct {
-	p layout.Path
+	p    layout.Path
+	tags []string
 }
 
 // NewLayout returns a new publish.Interface that saves images to an OCI Image Layout.
-func NewLayout(path string) (Interface, error) {
+func NewLayout(path string, tags []string) (Interface, error) {
```

Comment:
FYI this changes the method signature, and will break anyone that uses this as a library. I don't think this is a blocker, but if we expect that to be something people do more in the future we should be more careful.

Not a blocker this time though.

Reply:
We can change it as a variadic function, this does not affect people who might be using the ko project as a library.

Reply:
it should be good now